Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Use an official Node.js runtime as a parent image
+FROM node:22
+
+# Set the working directory in the container
+WORKDIR /usr/src/app
+
+# Create the cache directory
+RUN mkdir -p ./cache
+
+# Copy the current directory contents into the container at /usr/src/app
+COPY . .
+
+# Install any needed packages specified in package.json
+RUN npm install
+RUN npm update
```

Comment:
Copying the node packages from the host machine can be problematic (as in my case), because the format of the bettersql3 binary is not ELF.  Instead, let's just copy over the files we want and then have node install everything from scratch.

```suggestion
COPY .env *.js *.credentials package.json README.md LICENSE .

# Install any needed packages specified in package.json
RUN npm install && npm update
```